chore!: remove hipchat importer (#32154)
RocketChat/Rocket.Chat: .changeset/quiet-kings-rhyme.md

---
'@rocket.chat/meteor': major
---

> Removed the ability to import data in the HipChat Enterprise format, as it has been discontinued over five years ago.

Reviewer: ```suggestion
Removed the ability to import data in the HipChat Enterprise format, as it was discontinued over five years ago.
```
